nostr-java-event/src/main/java/nostr/event/Kind.java

```diff
@@ -1,53 +1,56 @@
-package nostr.event;

```

Comment:
lines [33-35](https://github.com/tcheeric/nostr-java/pull/156/files#diff-e82713367e1d9338e50239e1f89c0fe2c4bb1ad4550baacfe4a8b9ce5d690572R33-R35) are sole additions.  however, entire file contains hidden DOS CR/LF (^M) characters at EOL:

```
$ dos2unix -id nostr-java-event/src/main/java/nostr/event/Kind.java
53  nostr-java-event/src/main/java/nostr/event/Kind.java
```
git will manage line endings automatically as per line-endings configuration described on [GitHub](https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings?platform=windows)

Reply:
> lines [33-35](https://github.com/tcheeric/nostr-java/pull/156/files#diff-e82713367e1d9338e50239e1f89c0fe2c4bb1ad4550baacfe4a8b9ce5d690572R33-R35) are sole additions. however, entire file contains hidden DOS CR/LF (^M) characters at EOL:
> 
> ```
> $ dos2unix -id nostr-java-event/src/main/java/nostr/event/Kind.java
> 53  nostr-java-event/src/main/java/nostr/event/Kind.java
> ```
> 
> worth considering having git manage line endings automatically as per line-endings configuration described on [GitHub](https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings?platform=windows)

Noted, thank you